docs/en/sql-reference/sql-functions/Window_function.md

```diff
@@ -134,304 +165,256 @@ select stock_symbol, closing_date, closing_price,
+|    1 | lily  | english |   100 |   100 |
+|    2 | tom   | english |    98 |   100 |
+|    4 | amy   | english |    92 |   100 |
+|    5 | mike  | english |    85 |   100 |
+|    3 | jack  | english |  NULL |   100 |
+|    6 | amber | physics |   100 |   100 |
+|    3 | jack  | physics |    99 |   100 |
+|    4 | amy   | physics |    99 |   100 |
+|    5 | mike  | physics |    85 |   100 |
+|    1 | lily  | physics |    60 |   100 |
+|    2 | tom   | physics |  NULL |   100 |
+|    6 | amber | NULL    |    90 |    90 |
+|    3 | jack  | math    |    95 |    95 |
+|    6 | amber | math    |    92 |    95 |
+|    2 | tom   | math    |    80 |    95 |
+|    4 | amy   | math    |    80 |    95 |
+|    5 | mike  | math    |    70 |    95 |
+|    1 | lily  | math    |  NULL |    95 |
++------+-------+---------+-------+-------+
 ```
 
-As the above example shown, when `num_buckets` is `2`:
-
-* Rows of No.1 to No.6 were classified into the first partition; rows of No.1 to No.3 were stored in the first bucket, and rows of No.4 to No.6 were stored in the second one.
-* Rows of No.7 to No.9 were classified into the second partition; rows of No.7 and No.8 were stored in the first bucket, and row No.9 was stored in the second one.
-* Row No.10 was classified into the third partition and stored in the first bucket.
-
-### FIRST_VALUE()
+### LAST_VALUE()
 
-FIRST_VALUE() returns the **first** value of the window range.
+LAST_VALUE() returns the **last** value of the window range. It is the opposite of FIRST_VALUE().
 
-Syntax:
+**Syntax:**
 
 ```SQL
-FIRST_VALUE(expr [IGNORE NULLS]) OVER(partition_by_clause order_by_clause [window_clause])
+LAST_VALUE(expr [IGNORE NULLS]) OVER(partition_by_clause order_by_clause [window_clause])
 ```
 
-`IGNORE NULLS` is supported from v2.5.0. It is used to determine whether NULL values of `expr` are eliminated from the calculation. By default, NULL values are included, which means NULL is returned if the first value in the filtered result is NULL. If you specify IGNORE NULLS, the first non-null value in the filtered result is returned. If all the values are NULL, NULL is returned even if you specify IGNORE NULLS.
-
-Example:
-
-We have the following data:
+`IGNORE NULLS` is supported from v2.5.0. It is used to determine whether NULL values of `expr` are eliminated from the calculation. By default, NULL values are included, which means NULL is returned if the last value in the filtered result is NULL. If you specify IGNORE NULLS, the last non-null value in the filtered result is returned. If all the values are NULL, NULL is returned even if you specify IGNORE NULLS.
 
-```SQL
- select name, country, greeting
- from mail_merge;
- ```
+By default, last_value() calculates `rows between unbounded preceding and current row`, which compares the current row with all its preceding rows. If you want to show only one value for each partition, use `rows between unbounded preceding and unbounded following` after ORDER BY.
```

Comment:
LAST_VALUE()

Maybe we can use consistent letter cases for functions in the doc